ts/a11y/explorer/KeyExplorer.ts

```diff
@@ -295,6 +295,7 @@ export class SpeechExplorer extends AbstractExplorer<string> implements KeyExplo
     ['<', this.nextStyle.bind(this)],
     ['x', this.summary.bind(this)],
     ['-', this.expand.bind(this)],
+    ['d', this.depth.bind(this)],
```

Comment:
I'm wondering if this should be a symbol rather than a letter, like `=` or `#`, instead of a letter that ties to a word in a particular language.

Reply:
The problem with these symbols is that on many keyboard layouts they are only reachable via shift.
I will merge for now and then change it in the next PR.